README.md

```diff
@@ -33,10 +33,28 @@ import finite_state_sdk
 
 ## Generating the docs
 
+### Setting docs generation project
+
+`docs-generation` folder is the python project to generate html documentation.
+
+`./docs-generation/pyproject.toml` is the Poetry file project. It should run with python 3.9 or up.
```

Comment:
it says 3.8.1 in the pyproject

Reply:
there are 2 python projects in the repo, SDK runs with python 3.8.1 or up. To generate docs it have to run with python 3.9 or up. That` pyproject.toml` file  is in `docs-generation` folder